docs/view/Text/README.md

```diff
@@ -38,53 +36,13 @@ BpkText(context).apply {
 - `bpkFontFamilyEmphasized`
 - `bpkFontFamilyHeavy`
 
-- `bpkTextBaseAppearance`
```

Comment:
I removed these as we don't really want anybody to modify these anyways - happy to bring them back (but i would probably only put back the non-deprecated ones)